charts/kube-prometheus-stack/templates/grafana/dashboards-1.14/apiserver.yaml

```diff
@@ -1744,4 +1744,4 @@ data:
         "uid": "09ec8aa1e996d6ffcd6817bbaff4db1b",
         "version": 0
     }
-{{- end }}
+{{- end }}
```

Comment:
@kir4h Could you please retain the empty line at the end of the many files you've edited?

Reply:
If I'm not mistaken, by doing so I would "unsync" the script again (I have tested regenerating the dashboards from the CLI (to ensure no IDE is removing them) and there are no changes in git), as the empty line is not automatically generated.
It seems it was manually introduced in https://github.com/prometheus-community/helm-charts/pull/1044  (example: https://github.com/prometheus-community/helm-charts/pull/1044/files#diff-80b84898a52648f05e7a7caf69cf86be7fa57179f2ec9bc28f54d6e297a65305R1747)

Reply:
@kir4h You're right!